custom-metrics-stackdriver-adapter/Makefile

```diff
@@ -3,7 +3,7 @@ GOOS?=linux
 OUT_DIR?=build
 PACKAGE=github.com/GoogleCloudPlatform/k8s-stackdriver/custom-metrics-stackdriver-adapter
 PREFIX?=staging-k8s.gcr.io
-TAG = v0.12.2
+TAG = v0.13.0
```

Comment:
we're just changing the examples, why are you bumping the adapter's version?

Reply:
yesterday when we do the release, I notice that the image created was tagged to `v0.12.2` instead of `v0.13.0`, I can create a separate PR for this

Reply:
Sounds good → let's do it in a separate PR, since this is not related to this PR's fix.